saliency_model/itty_koch.py

```diff
@@ -1,53 +1,46 @@
 # development file, later on itti&koch class
-
 import math
 import numpy as np
 import scipy.signal as signal
-import matplotlib.pyplot as plt
 import matplotlib.image as mpimg
 
-import cv2
-# import saliency_model.utils
-from utils import receptiveFieldMatrix, mexicanHat, downsample_image
-from itti_koch_features import *
+from .utils import receptive_field_matrix, mexican_hat, downsample_image
```

Comment:
Not sure what those . do, for me it only works without them (if I run from the command line).